packages/api/src/feature/service.ts

```diff
@@ -0,0 +1,32 @@
+import createError from "http-errors";
+import { logger } from "@stela/logger";
+
+import { db } from "../database";
+
+import type { FeatureFlag, FeatureFlagRow } from "./models";
+
+const getFeatureFlags = async (): Promise<FeatureFlag[]> => {
+  const result = await db
+    .sql<FeatureFlagRow>("account.queries.get_feature_flags")
+    .catch((err) => {
+      logger.error(err);
+      throw new createError.InternalServerError(
+        "Failed to retrieve feature flags"
+      );
+    });
+
+  const featureFlags = result.rows.map<FeatureFlag>(
+    (row: FeatureFlagRow): FeatureFlag => ({
+      ...row,
+      id: row.id,
+      name: row.name,
+      globallyEnabled: row.globallyEnabled,
+    })
+  );
```

Comment:
We use code like this in other endpoints to transform some fields where the desired output is different from what's in the database, but I don't think there's a need for that here.